x-pack/docs/en/security/authorization/built-in-roles.asciidoc

```diff
@@ -183,8 +183,9 @@ However, this role does not grant permission to view the data in all indices.
 
 [[built-in-roles-watcher-admin]] `watcher_admin`::
 +
-Grants read access to the `.watches` index, read access to the watch history and
-the triggered watches index and allows to execute all watcher actions.
+Allows users to create and execute all watcher actions. Grants read access to
```

Comment:
AFAIK this product should be capitalized:
```suggestion
Allows users to create and execute all {watcher} actions. Grants read access to
```

Reply:
Thanks for catching this @lcawl